templates/blog/post_detail.html

```diff
@@ -0,0 +1,32 @@
+{% extends "base.html" %}
+{% load crispy_forms_filters %}
+
+{% block content %}
+  <h2>{{ post.title }}</h2>
+  <p>{{ post.owner }} at {{ post.created_time }}</p>
+  <p style="font-size: 20px">{{ post.content }}</p>
+  <p><b>{{ post.commentaries.count }} comment{{ post.commentaries.count|pluralize }}</b></p>
+  {% if post.commentaries.all %}
```

Comment:
use "if post.commentaries.exists"
because your solution firstly fetches all objects from db(for example 1 billion) and then checks if it isn't empty
but .exists checks the first record and if there is, it returns True, otherwise False